superset/commands/base.py

```diff
@@ -23,7 +24,7 @@ class BaseCommand(ABC):
     """
 
     @abstractmethod
-    def run(self):
+    def run(self) -> Any:
```

Comment:
What do you think about setting this to `Union[None, Model]`, will mypy complain?

Reply:
@dpgaspar `Union[None, Model]` is equivalent to `Optional[Model]`. I've made the change.